src/components/ForexRatesTab.js

```diff
@@ -9,25 +9,23 @@ import SnackbarContentWrapper from './SnackbarUtils';
 import { getForexRates } from '../api';
 
 export const stringRateFromDecimalRateAndInteger = (decimalRate, integer) => {
+  const forexRate = String(integer);
+  let precision = forexRate.length - decimalRate;
```

Comment:
You should be able to simplify allot of this logic doing something like this:

```
const forexRateWithPrecision = new MLNumber(`${integer}e-${precision}`) // can be instantiated with a string
return forexRateWithPrecision.toString()
```

Reply:
> You should be able to simplify allot of this logic doing something like this:
> 
> ```
> const forexRateWithPrecision = new MLNumber(`${integer}e-${precision}`) // can be instantiated with a string
> return forexRateWithPrecision.toString()
> ```

As discussed, I have added a `shiftedBy` method on [ml-number/releases/tag/v11.2.0](https://github.com/mojaloop/ml-number/releases/tag/v11.2.0), which can be used as follows:

```
const ratePrecision = 4
const shifted = new MLNumber('0123').shiftedBy(-1*ratePrecision)
return shifted.toString() <-- will return '0.0123'
```

Reply:
Thanks @mdebarros for adding the  'shiftedBy' function to @mojaloop/ml-number Library. 
Refactored the logic using the @mojaloop/ml-number Library